src/parse.c

```diff
@@ -164,11 +165,31 @@ xjwt__parse_ec_signature(xjwt_parsed_t *jwt, const char **outecsig,
   size_t offset = jwt->signature_decoded_l / 2;
   ECDSA_SIG *sig = ECDSA_SIG_new();
 
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
   /* TODO(pquerna): assert signature size constraints here? */
   BN_bin2bn((const unsigned char *)jwt->signature_decoded,
             jwt->signature_decoded_l / 2, sig->r);
   BN_bin2bn((const unsigned char *)jwt->signature_decoded + offset,
             jwt->signature_decoded_l / 2, sig->s);
+#else
+  BIGNUM *r, *s;
```

Comment:
can we  just use the new code path even with old OpenSSL?  I think so?  Then we'd not have a alternative code path based on version?